### ISSUE TITLE
Impossible to run in production mode

### ISSUE THREAD

**Issue by cv65kr** [jorge07/symfony-5-es-cqrs-boilerplate#171]
Before was exists possibility to run `make env=prod up`. 
Currently command failed due the bad structure of catalogs.
```
docker-compose -f docker-compose.yml -f etc/prod/docker-compose.yml up -d
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './etc/prod/docker-compose.yml'
make: *** [up] Error 1
```

**Comment by jorge07** [comment]
I use prod mode in minikube but I understand most people has other stacks. Let's fix this. Thanks for report it

**Comment by jorge07** [comment]
Have a try to #176 please

**Comment by jorge07** [comment]
solved in  #176
